src/explore-education-statistics-admin/src/services/types/content.ts

```diff
@@ -23,6 +23,8 @@ export interface Comment {
   setResolved?: boolean;
 }
 
+export type CommentType = 'inline' | 'block';
```

Comment:
It doesn't make sense for `CommentType` to be here in services as it's not relevant to this code.

t's only relevant to comments component code, so it'd make more sense for it to be exported from the `Comment` component. Alternatively, it could be in a separate file in a `components/comments/types` directory co-located with the rest of comments component code.

Reply:
I've moved it to the `Comment` component.